### ISSUE TITLE
Raising exception "Unknown node.func type for <ast.Call ...>"

### ISSUE THREAD

**Issue by andrei-shabanski** [miketheman/flake8-sqlalchemy#13]
I tried your linter and was faced with an exception. Thrown exception halts code checking, which appears not to be the correct behavior.

Do you have any further plans to change this behavior?

**Comment by miketheman** [comment]
Hi there! This is a very new library, and I haven't been testing it too extensively yet.

Can you share the conditions that raised this exception?

**Comment by andrei-shabanski** [comment]
flake8-sqlalchemy breaks on the following code

```python
from pydantic import BaseModel, validator


def foo(value):
    return value


class Model(BaseModel):
    title: str
    _normalize_title = validator("title", allow_reuse=True, always=True)(foo)
```

**Comment by miketheman** [comment]
@andrei-shabanski Thanks for the report - the fix should be out in version 0.2.0
